### PR TITLE
Rewrite grpc

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt [hunk 1]
```diff
@@ -95,15 +95,17 @@ class DeviceGatewayClient(policyResponse: PolicyResponse,
 
     override fun isConnected(): Boolean = isConnected
 
-    override fun send(request: MessageRequest) {
+    override fun send(request: MessageRequest) : Boolean {
         val event = eventStreamService
         val crash = crashReportService
 
         when(request) {
             is AttachmentMessageRequest -> event?.sendAttachmentMessage(toProtobufMessage(request))
```
Comment (shshshl09) [inline nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt]: I think that if event or crash is null, return false

File: nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/GrpcTransportFactory.kt [hunk 2]
```diff
@@ -1,24 +1,9 @@
-/**
```
Comment (shshshl09) [inline nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/GrpcTransportFactory.kt]: license removed?

File: nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt (hunk 1 above)
Comment (tommy5oo) [reply nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt]: Done

File: nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/GrpcTransportFactory.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/GrpcTransportFactory.kt]: Done

File: nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt (hunk 1 above)
Comment (tommy5oo) [reply nugu-client-kit/src/main/java/com/skt/nugu/sdk/client/port/transport/grpc/devicegateway/DeviceGatewayClient.kt]: Done, Change void to boolean!

**Pull request closed without merging**